cpp/arcticdb/util/buffer_pool.hpp

```diff
@@ -9,7 +9,11 @@
 
 #include <arcticdb/util/buffer.hpp>
 
-#include <third_party/recycle/src/recycle/shared_pool.hpp>
+#ifdef ARCTICDB_USING_CONDA
+    #include <recycle/shared_pool.hpp>
+#else
+    #include <third_party/recycle/src/recycle/shared_pool.hpp>
```

Comment:
Please use the include path

Reply:
Which include paths are you thinking about? Could you do a suggestion for what you are thinking about?

Reply:
[`ARCTICDB_USING_CONDA` has been introduced in #348 to distinguish between include paths](https://github.com/man-group/ArcticDB/pull/348/files#diff-9357e7e6906b72682dbae796b1a310682b3f7b334e3bba0cfee71c08f2e8cf57R444). 

As discussed, we should be able to modify the setup to rather have source agnostic include path, like this one:
```suggestion
    #include <recycle/shared_pool.hpp>
```